### ISSUE TITLE
ES9 is the tenth edition?

### ISSUE THREAD

**Issue by bergus** [tc39/ecma262#1489]
Hello,
I just found an editorial issue in the document published at https://www.ecma-international.org/ecma-262/9.0/. I hope this repo is the appropriate place to ask about that and possibly get it fixed.

Yes, I know about the offset between the year and the edition, but the [*Introduction* of above-mentioned document](http://ecma-international.org/ecma-262/9.0/#sec-intro) states

> This Ecma Standard […] is the tenth edition of the ECMAScript Language Specification. 

whereas the document is titled

> ECMA-262, 9th edition, June 2018

**Comment by ljharb** [comment]
https://en.wikipedia.org/wiki/ECMAScript#Versions shows and implies:
1. ES1
1. ES2
1. ES3
1. ES4 (Abandoned)
1. ES5
1. ES5.1
1. ES6 / ES2015
1. ES2016
1. ES2017
1. ES2018
1. ES2019

**Comment by nicolo-ribaudo** [comment]
@ljharb ES2017 (ES8) is reported as the 8th edition: https://ecma-international.org/ecma-262/8.0/; 2016 as the 7th and 2015 as th 6th

**Comment by ljharb** [comment]
The implication, then, is that the "9th edition" needs to be updated to "10th edition".

**Comment by nicolo-ribaudo** [comment]
What is the 9th edition then?

**Comment by jmdyck** [comment]
That's odd. Commit 4b219859 changed:
> This Ecma Standard defines the ECMAScript **2018** Language. It is the **ninth** edition...

to
> This Ecma Standard defines the ECMAScript **2019** Language. It is the **tenth** edition...

and yet https://www.ecma-international.org/ecma-262/9.0/ says:
> This Ecma Standard defines the ECMAScript **2018** Language. It is the **tenth** edition...

It's unclear how that combination could occur, but yes @bergus, "tenth" should be "ninth" there.

**Comment by jmdyck** [comment]
(BTW, note that 7a9a9b6b changed that line to:
> This Ecma Standard defines the ECMAScript **2020** Language. It is the **tenth** edition

i.e., didn't change "tenth" to "eleventh". That's a separate problem, but I point it out in case it misleads anyone.)

**Comment by bterlson** [comment]
Nice catch, I'll work with @ljharb to get that fixed up on the ECMA side.

**Comment by littledan** [comment]
Minor: Is this really the case, that ES 5.1 is the sixth edition? I thought ES6 was the sixth edition. Should we really be renumbering several past editions on the Ecma side?

**Comment by bergus** [comment]
@littledan No, ES 5.1 is not the sixth edition. I don't see how the Wikipedia article implies that anyway. The sixth edition is ES6 (ES2015).

**Comment by ljharb** [comment]
Fair point, i've updated my comment above.

**Comment by allenwb** [comment]
The edition of ECMA-262 published in June 2011 was intentionally and officially label as "Edition 5.1" on its front cover rather than as "6th Edition".

**Comment by allenwb** [comment]
The primary use of the "edition number" is to provide something that goes after "ECMA-262" when there is a need to write down the document id for a specific edition.  EG, ECMA-262-5.1.  In the ISO/ECMA/etc. standards world what goes in that trailing position is the edition number.

Skipping "Edition 4" and using "5.1" as an edition number were radical departures from Ecma conventions but we convinced Istvan that these made sense.  For the  "Harmony" release we changed the document title to include the publication year: "ECMAScript 2015 Language Specification" but did not take up the challenge of convincing Ecma that the "edition number" could also be the year.  EG, ECMA-262-2015.

I think it would be great if for future editions (even starting with 2019) we could switch to that convention.  But somebody would have to carry the ball of convincing the Ecma Secretariat to go along with the idea.  Based past experience with Editions 4 and 5.1 I think this might not be too hard to accomplish.
